File: social_club_PyTest/activity.py
class Activity:
    def __init__(self, new_name, new_place, new_start_date, new_cost):
        self.name = new_name
        self.place = new_place
        self.start_date = new_start_date
        self.cost = new_cost

    def __str__(self):
        return f'{self.name} starts on {self.start_date.strftime("%x")} at {self.place}'

File: Project_Files/Tests/JavaScript_SocialClub_Python_Bloodbank/model_answer/MochaChai/src/Activity.js
class Activity {
  constructor (newName, newPlace, newStartDate, newCost) {
    this.name = newName
    this.place = newPlace
    this.startDate = newStartDate
    this.cost = newCost
  }
  toString () {
    return `${this.name} starts on ${this.startDate.toLocaleDateString()} at ${this.place}`
  }
}
